internal/operator/operator.go

```diff
@@ -126,11 +127,13 @@ func NewOperator(o *OperatorOpts) (*operator, error) {
 		return nil, err
 	}
 
+	lfsl := logr.FromSlogHandler(l.Handler())
+	log.SetLogger(lfsl)
```

Comment:
occasionally, the operator will emit a stacktrace indicating that `log.SetLogger` hasn't been called - and thus, global logs aren't being emitted.  so, now we call `log.SetLogger`.